src/main/resources/templates/rooms/form.html

```diff
@@ -0,0 +1,48 @@
+<div class="row">
+        <div class="form-group col-md-6">
+          <label for="bname" class="col-form-label">First Name</label>
+          <input
+            type="text"
+            th:field="*{bname}"
+            class="form-control"
+            id="bname"
+            placeholder="First Name"
+          />
+          <span
+            th:if="${#fields.hasErrors('bname')}"
+            th:errors="*{bname}"
+            class="text-danger"
+          ></span>
+        </div>
+        <!-- <div class="form-group col-md-6">
```

Comment:
I'd delete commented code or uncomment it; it's usually a code smell to leave comments around.